samples/algorithms/database-search/Program.qs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.DatabaseSearch {
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Convert;
+
+    @EntryPoint()
+    operation RunDatabaseSearches() : Unit {
+        RunRandomSearch();
+        RunQuantumSearch();
+        RunMultipleQuantumSearch();
+    }
```

Comment:
Would this be a good place to demonstrate the multiple entry-point feature?

Reply:
I updated it to use multiple entry points now. Could you check if the changes to the readme are clear enough? (I left the IQ# version alone since it seems to define its own entry point.)